mobile/native/src/api.rs

```diff
@@ -248,6 +248,19 @@ pub fn run(
     config::set(config.clone());
     db::init_db(&app_dir, get_network())?;
     let runtime = ln_dlc::get_or_create_tokio_runtime()?;
+
+    // TODO: Hide this behind some debug flag, it shouldn't run in production
+    // (or maybe log once a minute in release build?)
+    let handle = runtime.handle();
+    let runtime_monitor = tokio_metrics::RuntimeMonitor::new(handle);
+    let frequency = Duration::from_secs(5);
+    runtime.spawn(async move {
+        for metrics in runtime_monitor.intervals() {
+            tracing::debug!(?metrics, "tokio metrics");
```

Comment:
What's the idea of how to use these metrics?

Reply:
it tells you all the information about the current runtime - descriptions of the metrics are here:

https://docs.rs/crate/tokio-metrics/latest

one of the most interesting for us are for example:
- [workers_count](https://docs.rs/tokio-metrics/0.2.*/tokio_metrics/struct.RuntimeMetrics.html#structfield.workers_count) - how many worker threads we actually have on our runtime (e.g. I was surprised seeing my iphone having 6!)
- https://docs.rs/tokio-metrics/0.2.*/tokio_metrics/struct.RuntimeMetrics.html#structfield.max_busy_duration - maximum amount the worker thread was busy. 

also, if the queue of tasks scheduled but not executed would be growing we could see some contention with things falling behind.

also:task metrics might be interesting too.

Reply:
> maximum amount the worker thread was busy

Interesting, does this include blocking threads?

Regardless, it will be good to make sure we aren't accidentally doing blocking work on the runtime!

Reply:
this is not intended to be merged; I have now made it clear by renaming the PR!

this is more of a showcase of another tool that one can have on their side branch.